test/test_structured_3d.jl

```diff
@@ -71,11 +71,17 @@ isdir(outdir) && rm(outdir, recursive=true)
       tspan = (0.0, 0.25))
   end
 
+  @trixi_testset "elixir_euler_cubed_sphere_coupled.jl" begin
+    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_cubed_sphere_coupled.jl"),
+      l2   = [0.004772751902799928, 0.004508606545076189, 0.004508606545076109, 0.004508606545076116, 0.008767829559836243],
+      linf = [0.037694511197162495, 0.04729357296587011, 0.04729357296588943, 0.030289012817550365, 0.08406230656858238])
+  end
+
   @trixi_testset "elixir_mhd_ec.jl" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_ec.jl"),
-      l2   = [0.009082353036644902, 0.007128360240528109, 0.006970330025996491, 0.006898850266874514,
-              0.03302008823756457, 0.003203389099143526, 0.003077498677885352, 0.0030740006760477624,
-              4.192129696970217e-5],
+      l2   = [0.009082355120108163, 0.00712836187575154, 0.006970331624968278, 0.0068988518494490746,
+              0.033020095812269036, 0.0032033898339909085, 0.0030774993838538736, 0.003074001381213843,
+              4.192130658631602e-5],
```

Comment:
Why would this change?

Reply:
We think it's because we are now calculating the total volume on the DGSEM nodes instead of the analysis nodes (which didn't change the errors of the other tests enough to make the tests fail), but we're not sure.